plugins/webhooks/src/modules/webhooks/service.ts

```diff
@@ -23,85 +24,79 @@ type ConstructorParams = {
   logger: Logger;
 };
 
+type WebhookOptions = LoaderOptions & { subscriptions: string[] } & { secretKey: string };
+
 class WebhooksService extends MedusaService({
   Webhook,
 }) {
   public subscriptions: string[] = [];
   private logger: Logger;
-
-  constructor(
-    container: ConstructorParams,
-    options: LoaderOptions & { subscriptions: string[] }
-  ) {
+  private options: WebhookOptions;
+  constructor(container: ConstructorParams, options: WebhookOptions) {
     super(container, options);
+    this.options = options;
     this.subscriptions = options.subscriptions;
+    if (!this.options.secretKey) {
+      this.logger.warn('No secretKey provided for webhook signatures. Webhook security will be compromised.');
```

Comment:
The wording here is a little extreme haha. Can we tweak this a bit?

No webhook secret key was found. Please provide a `secretKey` in the plugin options. A default will be used instead, but this may reduce the security of webhook signatures.